addons/hr_expense_stripe/models/hr_expense_stripe_credit_card.py

```diff
@@ -1,21 +1,22 @@
 import logging
 import re
 
-from odoo import api, fields, models
+from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError
+from odoo.addons.hr_expense_stripe.utils import make_request_stripe_proxy
 
 _logger = logging.getLogger(__name__)
-EXPIRATION_PATTERN = re.compile(r'\d{2}/\d{2}')  # 01/12
+EXPIRATION_PATTERN = re.compile(r'\d{2}/\d{4}')  # 12/2025
 
 # https://docs.stripe.com/api/issuing/cards
 class HrExpenseStripeCreditCard(models.Model):
     _name = 'hr.expense.stripe.credit.card'
     _inherit = ['mail.thread']
     _description = 'Employee Credit Card'
     _check_company_auto = True
-    _rec_name = 'last_4'
+    _rec_name = "card_name"
```

Comment:
```suggestion
    _rec_name = 'card_name'
```
Again, I don't think it's a good idea. I still prefer the names with the stars as they're less confusing and you understand what it's about